src/schemas/v2/combined.json

```diff
@@ -36814,6 +36814,182 @@
+                "type": "string"
+              },
+              "uprn": {
+                "title": "Uprn",
+                "type": "string"
+              },
+              "propertyType": {
+                "title": "Property Type",
+                "type": "string",
+                "enum": [
+                  "Apartment",
+                  "Flat",
+                  "Property",
+                  "Detached House",
+                  "Maisonette",
+                  "Semi-detached House",
+                  "Terraced House",
+                  "Others"
+                ]
+              },
+              "tenure": {
+                "title": "Ownership Type",
+                "description": "What type of ownership is the property?",
+                "type": "string",
+                "enum": [
+                  "Freehold",
+                  "Managed Freehold",
+                  "Commonhold",
+                  "Leasehold",
+                  "Other"
+                ]
+              },
+              "BasicDetails": {
+                "title": "Basic Property Details",
+                "type": "object",
+                "properties": {
+                  "bedrooms": {
+                    "title": "Number of bedrooms",
+                    "type": "integer"
+                  },
+                  "bathrooms": {
+                    "title": "Number of bathrooms",
+                    "type": "integer"
+                  },
+                  "yearBuilt": {
+                    "title": "Built Year",
+                    "type": "string",
+                    "format": "date"
+                  },
+                  "floorArea": {
+                    "title": "Floor area",
+                    "type": "object",
+                    "properties": {
+                      "inSquareMeter": {
+                        "type": "object",
+                        "properties": {
+                          "area": {
+                            "title": "Area in sqaure meters",
+                            "type": "number"
+                          },
+                          "pricePerSquareMeter": {
+                            "title": "Price per square meter",
+                            "type": "number"
+                          }
+                        }
+                      },
+                      "inSquareFoot": {
+                        "type": "object",
+                        "properties": {
+                          "area": {
+                            "title": "Area in sqaure foot",
+                            "type": "number"
+                          },
+                          "pricePerSquareFoot": {
+                            "title": "Price per square foot",
+                            "type": "number"
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              },
+              "sourceUrls": {
```

Comment:
this already exists as 'media' elsewhere in the schema so we should probably align with that or some abbreviated version of it

Reply:
used media as in main schema